pkg/controller/nodedaemon/daemonsets.go

```diff
@@ -87,6 +87,9 @@ func getDiskMakerDSMutateFn(
 		ds.Spec.Template.Spec.Containers[0].Image = common.GetDiskMakerImage()
 		ds.Spec.Template.Spec.Containers[0].Args = []string{"lv-manager"}
 
+		// to read /proc/1/mountinfo
+		ds.Spec.Template.Spec.HostPID = true
```

Comment:
Was existing code broken in https://github.com/openshift/local-storage-operator/pull/147 ? how did it pass before? cc @sp98

Reply:
That code only changed the discovery daemon, not the diskmaker daemon.

Reply:
@gnufied